Ready for Review: Wallet txs rpc

Patricia92 opened the pull request:
Fixes https://github.com/coblo/gui-demo/issues/134

Patricia92 committed:
Fix balance formatting

Patricia92 committed:
Add updater to get wallet transactions via rpc call

Patricia92 committed:
Add fetch more to wallet history

Patricia92 committed:
First fix for fetch more

Patricia92 committed:
Add handler for error in updater caused by race conditions

Patricia92 committed:
Insert new rows at top and remove unconfirmed transactions

titusz commented:
Something is not right with the updating logic. After having the wallet open for a while it looks like this:
![image](https://user-images.githubusercontent.com/278556/36320814-cdb63b06-1347-11e8-9dda-3a5769b61ff5.png)

Patricia92 committed:
Fix balance not updating

We still have issues with getbalance; if the user mines a block, getbalance does not consider this until the next block is mined.
